src/client/lazy-app/Compress/Options/style.css

```diff
@@ -118,6 +128,11 @@
   svg {
     fill: var(--header-text-color);
   }
+
+  &:focus {
```

Comment:
can you also add this new style to the new buttons (merged last week) `.save-button, .import-button`? a little akward atm that one has it and the other doesnt.

<img width="125" alt="Screenshot 2023-04-11 at 12 23 12 PM" src="https://user-images.githubusercontent.com/1134620/231267583-60ce78c7-1bfa-4afe-abe5-b83cdf761d0f.png">

Reply:
Ahh nice catch, didn't see the new icons. I'll apply the same style to them.